tap_typeform/client.py

```diff
@@ -138,7 +189,7 @@ def get_page_size(self, config):
         if page_size is None:
             return
         if ((type(page_size) == int or type(page_size) == float) and (page_size > 0)) or \
-            (type(page_size) == str and page_size.replace('.', '', 1).isdigit() and (float(page_size) > 0) ):
+                (type(page_size) == str and page_size.replace('.', '', 1).isdigit() and (float(page_size) > 0)):
```

Comment:
we can simplify this by a small block instead of using a function to parse page_size in the init
eg:
```python
try:
    page_size = int(config.get('page_size') or 0)
    if page_size > 0:
        self.page_size = page_size
        self.form_page_size = min(self.form_page_size, self.page_size)
    else:
        raise ValueError("Negative or ")
 except (ValueError, TypeError) as err:
    LOGGER.info("Invalid Value: %s, for page_size", config.get('email_activity_date_window', 0))
    raise from err
```

Reply:
This is unintentional refactoring change without any logical change. I would not prefer to cause any regression because of new changes.

Reply:
okay